demos/tests/cases.py

```diff
@@ -307,17 +306,18 @@ def device_cases(*args):
     )),
 
     # TODO: object_detection_demo_yolov3_async: no models.lst
-
-    PythonDemo(subdirectory='segmentation_demo', test_cases=combine_cases(
```

Comment:
You seem to have accidentally erased the `segmentation_demo` test cases. Please put them back.

Reply:
done